plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -467,11 +471,18 @@ jQuery( function ( $ ) {
 
 			attribute_row_indexes();
 
-			$attributes
+			const lastAttribute = $attributes
 				.find( '.woocommerce_attribute' )
-				.last()
-				.find( 'h3' )
-				.trigger( 'click' );
+				.last();
+
+			lastAttribute.find( 'h3' ).trigger( 'click' );
+
+			if ( $attributes.length === 1 ) {
+				// if there is only one attribute, show the actions that were hidden when user finishes typing attribute value to avoid a roundtrip
+				lastAttribute.find( '[name^="attribute_values"]' ).on( 'blur', function () {
+					$( '#product_attributes_actions' ).removeClass( 'hidden' );
+				} );
+			}
```

Comment:
Should we do this when `$attributes.length > 0` to be a bit more robust and handle some unexpected case somehow more than one attribute was added?